auth/auth.go

```diff
@@ -35,26 +35,28 @@ type Guards map[string]*Guard
 type Auth struct {
 	cache  cache.Cache
 	config config.Config
+	ctx    http.Context
 	guard  string
 	orm    orm.Orm
 }
 
-func NewAuth(guard string, cache cache.Cache, config config.Config, orm orm.Orm) *Auth {
+func NewAuth(guard string, cache cache.Cache, config config.Config, ctx http.Context, orm orm.Orm) *Auth {
```

Comment:
`ctx` is usually in the first parameter.

```suggestion
func NewAuth(ctx http.Context, guard string, cache cache.Cache, config config.Config, orm orm.Orm) *Auth {
```